config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
-  devise_for :users
-  # root "home#index"
+  get 'home/index'
+  #devise_for :users
```

Comment:
remove this unnecessary comment

Reply:
This is noted Ma'am, next time, I will remove unnecessary comment, thank you for reminding me